src/ratings-and-reviews/index.jsx

```diff
@@ -18,25 +18,107 @@ const reviewsTotal = (reviews) => {
   return total;
 };
 
-const RatingsAndReviews = (/* { reviews, metadata } */) => (
-  <div id="ratings-and-reviews">
-    <div className="column-1">
-      <h2 className="reviews-header">Ratings & Reviews</h2>
-      <span className="reviews-rating">{metadata.rating.toFixed(1)}</span>
-      <div>
-        <StarRating rating={metadata.rating}/>
+/** @param {Object} reviews*/
+const reviewsAverage = (reviews) => {
+  let total = 0;
+  let count = 0;
+  for (let i in reviews) {
+    total += i * reviews[i];
+    count += reviews[i];
+  }
+  let average = (total / count).toFixed(1);
+  return Number.parseFloat(average);
```

Comment:
You forgot our whole conversation! Don't do any of this. There are lots of things wrong with this approach, which I can go over again if you want, but the important point is you don't need to do any of this. In fact, you don't need a `reviewsAverage` function at all. `metadata.rating` is already the average of the reviews. You don't need to do anything. The number is already there. You can see it being used in the old version of this file before this commit.

Reply:
Yea I def missed that property.